### DISCUSSION TITLE
delivery reports

### DISCUSSION THREAD

**Discussion by krzysiekhelp** [Salamek/huawei-lte-api#179] [General]
how to associate sms with delivery reports. I have the only number to call, but it's not enough

 raport success

{'Smstat': '0', 'Index': '40006', 'Phone': '+48508xxxx', 'Content': None, 'Date': '2023-04-02 19:13:06', 'Sca': None, 'SaveType': '0', 'Priority': '0', 'SmsType': '7'}

sms

 {'Smstat': '3', 'Index': '40005', 'Phone': '+48508xxxxx', 'Content': 'test', 'Date': '2023-04-02 19:13:01', 'Sca': None, 'SaveType': '3', 'Priority': '4', 'SmsType': '2'},

what are the possibilities and what does 'Smstat', 'Sca', 'SmsType'  mean?

**Comment by Salamek** [comment]
SCA is  Message center number in INTL format eg. +420603052000
smstat is https://github.com/Salamek/huawei-lte-api/blob/8c8b269e36c8f63d7fdb7343e1848c5ddc3cdf64/huawei_lte_api/enums/sms.py#L59

SmsType is https://github.com/Salamek/huawei-lte-api/blob/8c8b269e36c8f63d7fdb7343e1848c5ddc3cdf64/huawei_lte_api/enums/sms.py#L50

Only way to pair sms reports i know is to pair by number and time, that is it, idk about any UUID or so

**Comment by krzysiekhelp** [comment]
thank you, very helpful

pozdrawiam
Krzysztof Siwiec


niedz., 2 kwi 2023 o 22:12 Adam Schubert ***@***.***>
napisał(a):
